server/src/main/java/org/elasticsearch/http/HttpTracer.java

```diff
@@ -57,12 +57,13 @@ HttpTracer maybeLogRequest(RestRequest restRequest, @Nullable Exception e) {
         if (logger.isTraceEnabled() && TransportService.shouldTraceAction(restRequest.uri(), tracerLogInclude, tracerLogExclude)) {
             logger.trace(
                 () -> format(
-                    "[%s][%s][%s][%s] received request from [%s]",
+                    "[%s][%s][%s][%s] received request from [%s]%s",
```

Comment:
should the new %s be also in square brackets?

Reply:
I was aiming to get it looking as similar as possible to the response log format

Reply:
I wonder if we actually don't want to keep this the same in both request and response. so that it would be easier to parse. the same as it is done for x-opaque-id. That means that trace.id or x-opaque-id are duplicated in a response. Present in both message and its own field.
I leave that up to you, or feel free to bring it up with the team

Reply:
Getting the trace id in there is not completely simple, so I would prefer to leave this hack as small a footprint as possible - if people do complain about the difference, we can then spend more time on this looking at it properly